buildSrc/src/main/kotlin/io/opentelemetry/gradle/OtelBomExtension.kt

```diff
@@ -6,9 +6,15 @@
 package io.opentelemetry.gradle
 
 import org.gradle.api.Project
+import org.gradle.api.model.ObjectFactory
 import org.gradle.api.provider.Property
 import java.util.function.Predicate
 
 abstract class OtelBomExtension {
   abstract val projectFilter: Property<Predicate<Project>>
+  var fallbacks: MutableSet<String> = hashSetOf()
```

Comment:
Nit: the set itself is mutable, so the field can use `val`
```suggestion
  val fallbacks: MutableSet<String> = hashSetOf()
```